addons/web_settings_dashboard/static/tests/dashboard_tests.js

```diff
@@ -0,0 +1,64 @@
+odoo.define('web_settings_dashboard.dashboard_tests', function (require) {
+"use strict";
+
+var testUtils = require('web.test_utils');
+var Dashboard = require('web_settings_dashboard');
+var Widget = require('web.Widget');
+
+QUnit.module('Settings', {}, function () {
+
+    QUnit.test('Dashboard: Invite new user', function (assert) {
+        assert.expect(1);
+        var self = this;
+
+        function createParent(params) {
+            var widget = new Dashboard.Dashboard();
+            testUtils.addMockEnvironment(widget,{
+                mockRPC: function (route, args) {
+                    if (route === '/web_settings_dashboard/data') {
+                        return $.when(params['data']);
+                }
+                return this._super(route, args);
+                },
+            });
+            return widget;
+        }
+
+        this.data = {
+            'active_users': 1,
+            'pending_counts': 1,
+            'pending_users':[{'id': 1, 'email': 'xyz@odoo.com'}],
+        }
+
+        var parent = createParent({
+            'data': {'apps':{},
+                    'share':{},
+                    'users_info': self.data,
+                    }
+        });
```

Comment:
Why do you need all this tinkering with createParent ? Can't you just define the data object above then do var parent = new Dashboard.Dashboard(); then have the mockRPC directly?